src/main/java/seedu/address/ui/BirthdayReminderCard.java

```diff
@@ -0,0 +1,62 @@
+package seedu.address.ui;
+
+import java.util.Optional;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.Region;
+import seedu.address.model.person.Birthday;
+import seedu.address.model.person.Person;
+
+public class BirthdayReminderCard extends UiPart<Region> {
+    private static final String FXML = "BirthdayReminderListCard.fxml";
+
+    /**
+     * Note: Certain keywords such as "location" and "resources" are reserved keywords in JavaFX.
+     * As a consequence, UI elements' variable names cannot be set to such keywords
+     * or an exception will be thrown by JavaFX during runtime.
+     *
+     * @see <a href="https://github.com/se-edu/addressbook-level4/issues/336">The issue on AddressBook level 4</a>
+     */
+
+    public final Person person;
+
+    @javafx.fxml.FXML
+    private HBox cardPane;
+    @FXML
+    private Label name;
+    @FXML
+    private Label phone;
+    @FXML
+    private Label birthday;
+
+    /**
+     * Creates a {@code PersonCode} with the given {@code Person} and index to display.
+     */
+    public BirthdayReminderCard(Person person) {
+        super(FXML);
+        this.person = person;
+        name.setText(person.getName().fullName);
+        phone.setText(person.getPhone().value);
```

Comment:
Hmmm should we add a getFullName() and getPhoneNumber() methods for person? Because this is currently violating law of demeter (?). Or should we just ignore